README.md

```diff
@@ -22,10 +22,24 @@ To complete this workshop you will require the following:
     * If you don’t already have an AWS account, create one at <https://aws.amazon.com> by following the on-screen instructions
 * Your access to the AWS account must have IAM permissions to launch AWS CloudFormation templates that create IAM roles.
 
+## Contents
+
 ### [Step 0 - Deploy the Cloudformation Stacks](docs/step-0.md)
+
+Step 0 will guide you through the deployment of the templates required for the workshop.
+
 ### [Step 1 - Getting Started with AWS WAF Security Automations Solution](docs/step-1.md)
+
+Step 1 will set up the WAF with your application. It will explore how the WAF Security Automation Solution works to block three types of attack.
```

Comment:
Can we mention these attacks here? e.g. (SQL Injection, X-site scripting...)

Reply:
Done